docker-compose.yml

```diff
@@ -16,6 +16,17 @@ services:
     depends_on:
       - postgres
 
+  client:
+    build:
+      context: client
+    volumes:
+        - ./client:/srv/app
+        - /srv/app/build
+        - /srv/app/node_modules
```

Comment:
Stops `./client:/srv/app` from butchering the install, but an anonymous volume is created so `docker-compose down -v` will be needed on a change.